packages/frontend/src/components/SettingsValidator/ValidatorTable.tsx

```diff
@@ -51,15 +52,28 @@ export const ValidatorTable: FC<{
     data: ValidationResponse[];
     projectUuid: string;
 }> = ({ data, projectUuid }) => {
-    const history = useHistory();
     const { classes } = useTableStyles();
+    const { colors } = useMantineTheme();
+
+    const history = useHistory();
+    const location = useLocation<{ validationId: number }>();
+    const searchParams = new URLSearchParams(location.search);
+    const validationId = searchParams.get('validationId');
+
+    const refs = useMemo(
+        () =>
+            data.reduce((acc, value) => {
+                acc[value.validationId.toString()] = createRef();
+                return acc;
+            }, {} as { [key: string]: RefObject<HTMLTableRowElement> }),
+        [data],
+    );
+
+    useScrollAndHighlight(refs, validationId, colors);
 
     const handleOnValidationErrorClick = (
         validationError: ValidationResponse,
     ) => {
-        if (!validationError.chartUuid && !validationError.dashboardUuid)
-            return null;
-
```

Comment:
Don't we still need this check for table errors?

Reply:
```
const getLinkToResource = (
    validationError: ValidationResponse,
    projectUuid: string,
) => {
    if (validationError.chartUuid)
        return `/projects/${projectUuid}/saved/${validationError.chartUuid}`;
    if (validationError.dashboardUuid) {
        return `/projects/${projectUuid}/dashboards/${validationError.dashboardUuid}/view`;
    }

    return;
};
 
```
returns `undefined` for non-charts/non-dashboards

so this will be a no-op for Tables:

```
   const handleOnValidationErrorClick = (
        validationError: ValidationResponse,
    ) => {
        const link = getLinkToResource(validationError, projectUuid);
        if (link) history.push(link);
    };

```